File: python/01_installable_script/source/package/application.py
${{VAR_COPYRIGHT_HEADER}}
"""Installable script main entry point."""

import sys


def main(argc: int, argv: list[str]) -> int:
    """Main function of the ${{VAR_PROJECT_NAME}} application.

    Args:
        argc (int): The argument count.
        argv (list[str]): The list of program arguments.

    Returns:
        int: The exit code of the program.
    """
    print("${{VAR_PROJECT_SLOGAN_STRING}}")
    if argc > 1:
        print("Args: " + str(argv[1:]))

    return 0


if __name__ == "__main__":
    sys.exit(main(len(sys.argv), sys.argv))

File: cpp/01_executable/source/src/main/cpp/namespace/Application.cpp
${{VAR_COPYRIGHT_HEADER}}

#include <string>

#include "${{VAR_NAMESPACE_PATH}}/Application.h"

${{VAR_NAMESPACE_DECL_BEGIN}}
using std::string;

string Application::getValue() {
    return "${{VAR_PROJECT_SLOGAN_STRING}}";
}

${{VAR_NAMESPACE_DECL_END}}
